### PR TITLE
New translations, including v1 of all dex entries

### PR DESCRIPTION
**Pull request by basvroegop**

### CODE REVIEW

File: maps/Route36NationalParkGate.asm [hunk 1]
```diff
@@ -474,378 +474,382 @@ Route36NationalParkGatePlayerWaitWithContestantsMovement:
...
+	text "Het is @" ; "Today's @"
 	text_ram wStringBuffer3 ; Day of the week MaxLength 9
 	text "."
-	line "That means the"
+	line "De dag van het" ; "That means the"
 
-	para "Bug-Catching Con-"
-	line "test is on today."
+	para "wekelijkse" ; "Bug-Catching Con-"
+	line "KEVERTOERNOOI." "test is on today."
```
Comment (wfowler1) [inline maps/Route36NationalParkGate.asm:483]: kapot

File: maps/Route36NationalParkGate.asm [hunk 2]
```diff
@@ -474,378 +474,382 @@ Route36NationalParkGatePlayerWaitWithContestantsMovement:
...
+	line "kunt vinden!" ; "find!"
 	done
 
 Route36NationalParkGateOfficer1AskToUseFirstMonText:
 	text "Uh-oh…"
 
-	para "You have more than"
-	line "one #MON."
+	para "Je hebt meer dan" ; "You have more than"
+	line "één #MON." "one #MON."
```
Comment (wfowler1) [inline maps/Route36NationalParkGate.asm:537]: here too

**Comment by wfowler1** [comment]
OH those maps got put into data/maps instead of maps/

File: data/text/battle.asm [hunk 3]
```diff
@@ -352,7 +352,7 @@ BattleText_TargetsEncoreEnded:
 BattleText_StringBuffer1GrewToLevel:
 	text_ram wStringBuffer1 ; MaxLength MaxPokemonNameLength
 	text " groeide" ; " grew to"
-	line "naar level @"
+	line "naar niveau @"
```
[the remark this answers is not in the document]
Comment (basvroegop) [reply data/text/battle.asm:355]: I somehow assumed we went for 'niveau', but stat summary page also has level...

We do have a LEVEL BALL called NIVEAUBAL, but that's our own translation. I'll try and get it consistent when done with work later this week.

**Pull request merged**